source/simulator/stokes_matrix_free.cc

```diff
@@ -744,7 +765,20 @@ namespace aspect
                                                           velocity.get_symmetric_gradient (q);
             VectorizedArray<number> pres = pressure.get_value(q);
             VectorizedArray<number> div = trace(sym_grad_u);
-            pressure.submit_value(-pressure_scaling*div, q);
+
+            if (strain_rate_table !=nullptr
+                && !(strain_rate_table->empty()))
+              {
+                // Note that derivative_scaling_factor is multiplied to viscosity_derivative_wrt_pressure_table
```

Comment:
Do you mean:
```suggestion
                // Note that derivative_scaling_factor has already been multiplied to viscosity_derivative_wrt_pressure_table
```
Because I do not see it happening here.